### ISSUE TITLE
allow selecting fixed payment token type in circle settings

### ISSUE THREAD

**Issue by levity** [coordinape/coordinape#808]
- also have a "Disabled" option--this is the default
- autocomplete field that includes the token types from any existing vaults

Depends on #888

**Comment by sshmm** [comment]
@levity Can I work on this?

**Comment by levity** [comment]
sure!

**Comment by sshmm** [comment]
@topocount  I was working on it. did you start working on it?

**Comment by levity** [comment]
@sshmm how's it going?

**Comment by levity** [comment]
@sshmm there have been a lot of changes in the frontend recently (e.g. #800 and #856). if you have some WIP, please push a draft request so we can see if there are any conflicts

**Comment by sshmm** [comment]
@levity  I had several health issues in the past two weeks so I didn't do much. I replaced the AdminCircleModal and its subcomponents with stitches components and still working on the styling and the behavior with use-form then add the fixed payment option. 
if @topocount already started it and made better achievement, I can pick another issue.

**Comment by sshmm** [comment]
> No worries. How about you push a draft PR when you have a chance, and then @topocount can consider building on top of it so your work doesn't go to waste?

ok, I will update it to an acceptable state and push it
